nixos/tests/cross-seed.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, ... }:
+let
+  apiKey = "twentyfourcharacterskey!";
+in
+{
+  name = "cross-seed";
+  meta.maintainers = with lib.maintainers; [ pta2002 ];
+
+  nodes.machine =
+    { pkgs, config, ... }:
+    let
+      cfg = config.services.cross-seed;
+    in
+    {
+      systemd.tmpfiles.settings."0-cross-seed-test"."${cfg.settings.torrentDir}".d = {
+        inherit (cfg) user group;
+        mode = "700";
+      };
+
+      services.cross-seed = {
+        enable = true;
+        settings = {
+          outputDir = "/var/lib/cross-seed/output";
```

Comment:
This is the default right? Maybe we should not set it to emphasize that we are testing with only the minimal set of required options?